packages/app/src/pages/landelijk/gedrag.tsx

```diff
@@ -119,3 +182,12 @@ const BehaviorPage: FCWithLayout<typeof getStaticProps> = ({
 BehaviorPage.getLayout = getNationalLayout;
 
 export default BehaviorPage;
+
+const IconContainer = styled.span(
+  css({
+    marginRight: 3,
+    color: 'gray',
+    height: '25px',
+    width: '25px',
```

Comment:
```suggestion
    height: 25,
    width: 25,
```

or maybe even this will work with styled-system:
```suggestion
    size: 25,
```